kompari/src/imgdiff.rs

```diff
@@ -15,7 +15,8 @@ pub enum ImageDifference {
     Content {
         n_different_pixels: u64,
         distance_sum: u64,
-        diff_image: Image,
+        rg_diff_image: Image,
+        overlay_diff_image: Image,
```

Comment:
This work always being done twice concerns me slightly for performance reasons. I think the pattern I'd like to see is:
1) only the absolute/scalar difference is calculated.
2) producing the diff images is a separate step, either with a function for each or a "mode" enum.